src/sys/wait.rs

```diff
@@ -215,6 +215,22 @@ fn decode(pid : Pid, status: i32) -> WaitStatus {
     }
 }
 
+/// Counterpart of the POSIX `waitpid` function
```

Comment:
This should actually describe what `waitpid` does with a brief summary line at the top and maybe an extended summary below it (with a blank comment line in between). Just saying it is a wrapper around `waitpid` isn't very helpful.

Additionally can you link to [the man page](http://man7.org/linux/man-pages/man2/waitpid.2.html) in the extended description here.